docs/components.adoc

```diff
@@ -0,0 +1,173 @@
+
+=== Impact when down
+* Emails are not sent
+* Communication with `backend-listener` breaks: changes in Admin Portal will not propagate to Backend
+* Backend alerts will not be triggered
+* Webhooks will not be triggered
+* Zync will not receive any updates
+* Background jobs will not be processed and will start accumulating in `system-redis`
+
+== system-mysql
+
+=== Responsibilities
+* It is the DB used by `system-app`
+
+=== Impact when down
+* `system-app` stops working
+
+== system-redis
+
+=== Responsibilities
+* It is the DB used by `system-app` as a queue for sidekiq jobs processed by `system-sidekiq`
+
+=== Impact when down
+* Background jobs are not stored
+* Billing does not run
+
+== system-memcache
+
+=== Responsibilities
+* Memcached is an ephemeral cache of values used to speed-up the performance of the `system-app` web application
+
+=== Impact when down
+* `system-app` misses its cache (mostly used by the UI to speed-up page loading but also by billing), making `system-app` to run slightly slower while the cache is rebuilt
+
+== system-sphinx
+
+=== Responsibilities
+* Fulltext search for `system-app` (caching some `system-mysql` content)
+
+=== Impact when down
+* The search functionality on the `system-app` Admin/Developer Portal (accounts and proxy rules search bars, templates, forum searches...) stops working
+
+== zync
+
+=== Responsibilities
+
+* Receives events from `system-sidekiq`
+* Enqueue those events as new jobs to be processed in the background by `zync-que`
+* Those enqueued jobs can be:
+- Creation/Update of OpenShift Routes (Admin/Developer portals of each tenant)
+- Creation/Update of OpenShift Routes (`apicast-staging` or `apicast-production` domains of each API)
+- Synchronization of information with configured 3rd party IDPs
+
+
+=== Impact when down
+* New jobs for OpenShift Routes creation or modification to `apicast-staging` and `apicast-production` cannot be created
+* New jobs for OpenShift Route creation or modification to Admin Portal and Developer Portal domains for new tenants cannot be created
+* New jobs for creation or Updates of Application (IDP) configuration cannot be created
+* `system-sidekiq` will retry the failed requests for some time
+
+== zync-que
+
+=== Responsibilities
+* Process the enqueued jobs created by `zync`
+* Those jobs can be:
+- Creation/Update of OpenShift Routes (Admin/Developer portals of each tenant)
+- Creation/Update of OpenShift Routes (`apicast-staging` or `apicast-production` domains of each API)
+- Synchronization of information with configured 3rd party IDPs
+
+=== Impact when down
+* Routes to `apicast-staging` or `apicast-production` cannot be created or updated
+* New OpenShift Routes to Admin Portal and Developer Portal domains for new tenants cannot be created
+* Creation or Updates of Application configuration are not synchronised to 3rd Party IdPs
+* Jobs will not be processed and will start accumulating in `zync-database`
```

Comment:
```suggestion
* Jobs will not be processed and will start accumulating in `zync-database`, which could lead to database out of memory related problems
```

Reply:
Fixed https://github.com/3scale/3scale-Operations/commit/0f4f10309441d9abc1a2c44ce377a6a7ce4bd84c, on that case is disk space not memory because it is postgres